conjurapi/host_factory_test.go

```diff
@@ -30,7 +30,7 @@ func TestClient_Token(t *testing.T) {
 		{
 			name:        "Create a token",
 			duration:    "10m",
-			hostFactory: "cucumber:host_factory:factory",
```

Comment:
Instead of randomly tweaking existing tests to use a partial ID, could we add a new one that explicitly tests that a partial ID works?

Reply:
yes, that's better